### ISSUE TITLE
Cannot read property 'jQuery' of undefined

### ISSUE THREAD

**Issue by zomars** [leafo/sticky-kit#200]
I'm getting this error when the file is being concatenated in a minified file and it's pointing to [this line](https://github.com/leafo/sticky-kit/blob/master/dist/sticky-kit.js#L10).

**Comment by dmitrylebedev** [comment]
Hi, I delete _temporary_ this from string `this.jQuery` and left this:
```javascript
$ = window.jQuery;

**Comment by zomars** [comment]
Tha's what I did too.

**Comment by leafo** [comment]
Is this something about your setup that would have jQuery not be available in the name jQuery?

**Comment by zomars** [comment]
jQuery is available, but not under the "this" namespace. If it helps to know, I'm using the jquery dist file from bower. And the only thing I'm doing is concatenating those files in the correct order.

**Comment by leafo** [comment]
`this` should default to `window`, is the library getting wrapped in something that's changing that?

**Comment by tim-field** [comment]
Yeah can confirm that `this.jQuery` is causing this module to be unusable with a modern build system ( browserify or webpack )   Can it just be removed ?

**Comment by zomars** [comment]
It's not getting wrapped and it's the first file being called.

**Comment by rebusweb** [comment]
@leafo could you please update npm package for v1.3 ?

**Comment by maurobringolf** [comment]
Am I missing something or is this still a thing? I installed the lib via npm (1.1.3 on npm), but still have the `this.jQuery` in it. I see that the current version here on GitHub has it fixed, but still get the old code and error from npm.

**Comment by rebusweb** [comment]
@maurobringolf Don't use npm package, use npm and github link `github:leafo/sticky-kit`

**Comment by maurobringolf** [comment]
@rebusweb Will this install master branch or latest release on github? Thanks for the hint! But this is a workaround, right?

**Comment by rebusweb** [comment]
@maurobringolf it's master, but you can also use `github:username/package#commit` convention.
yes, it's kind of a workaround, but using github links in npm is a standard functionality.

**Comment by maurobringolf** [comment]
Awesome, thank you very much!

**Comment by OnkelTem** [comment]
How do you guys manage to build it?
I get another error: https://github.com/leafo/sticky-kit/issues/248

@rebusweb How do you include it? Do you use Browserify?

**Comment by rebusweb** [comment]
@OnkelTem nope.
